src/components/PlayListItem/PlayListItem.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import playlists from "../../assets/Playlists.json";
+import "./PlayListItem.scss";
+
+const PlayListItem = () => {
+  return playlists.playlists.items.map(({ external_urls, id, name }) => {
```

Comment:
move the map inside the parent component. just u did for other components.

Reply:
Done.